### PR TITLE
feat(config): Initial Nice IBT4 BusT4 

### PR DESCRIPTION
**Pull request by cdnninja**
Started on slot one.

### CODE REVIEW

File: packages/config/config/devices/0x0441/ibt4zwave.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+					"value": 1
+				},
+				{
+					"label": "CO Alarm",
+					"value": 2
+				},
+				{
+					"label": "Heat Alarm",
+					"value": 4
+				},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0441/ibt4zwave.json]: Missing CO2 alarm (3): https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/notifications.json#L129

File: packages/config/config/devices/0x0441/ibt4zwave.json [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+			"productId": "0x1000",
+			"zwaveAllianceId": 3837
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"30[0xff]": {
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0441/ibt4zwave.json]: The bitmasks are reversed. See https://github.com/zwave-js/node-zwave-js/pull/2525#issuecomment-832466839

File: packages/config/config/devices/0x0441/ibt4zwave.json [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+					"value": 21
+				},
+				{
+					"label": "Home Monitoring",
+					"value": 22
+				}
+			]
+		},
+    "30[0xff00]": {
+			"label": "Alarm Configuration - 1st Slot Notification Status",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0441/ibt4zwave.json]: Did you see https://github.com/zwave-js/node-zwave-js/pull/2525#issuecomment-832471286 ?
I _think_ this _should_ be the Notification Event (with 255 meaning "Any Event", whatever that does 🤷🏻‍♂️)
At least if we trust the explanation in the manual.

File: packages/config/config/devices/0x0441/ibt4zwave.json [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+					"value": 255
+				},
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+        "30[0xff0000]": {
+			"label": "Alarm Configuration - 1st Slot Notification Event",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0441/ibt4zwave.json]: Dito, this probably should be "Notification Event Parameter"

File: packages/config/config/devices/0x0441/ibt4zwave.json [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+					"value": 1
+				},
+				{
+					"label": "Close",
+					"value": 2
+				}
+			]
+		},
+		"31": {
+			"label": "Alarm Configuration - 2nd Slot",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0441/ibt4zwave.json]: Once we have Slot 1 correct, you can just `$import` the parameters from slot 1 and change the labels.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0441/ibt4zwave.json (hunk 5 above)
Comment (cdnninja) [reply packages/config/config/devices/0x0441/ibt4zwave.json]: Perfect - I will take a look at that. Only difference thought is defaults for each slot.

Comment (AlCalzone) [reply packages/config/config/devices/0x0441/ibt4zwave.json]: You can override that too.

File: packages/config/config/devices/0x0441/ibt4zwave.json (hunk 4 above)
Comment (cdnninja) [reply packages/config/config/devices/0x0441/ibt4zwave.json]: Fixed

File: packages/config/config/devices/0x0441/ibt4zwave.json (hunk 5 above)
Comment (cdnninja) [reply packages/config/config/devices/0x0441/ibt4zwave.json]: Done.  Couldn't find a perfect example of importing from within the same file.  Let me know if syntax is wrong.

File: packages/config/config/devices/0x0441/ibt4zwave.json [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+					"value": 1
+				},
+				{
+					"label": "Close",
+					"value": 2
+				}
+			]
+		},
+		"31[0xff000000]": {
+			"$import": "#30[0xff000000]",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0441/ibt4zwave.json]:
```suggestion
			"$import": "#paramInformation/30[0xff000000]",
```
same for the other ones.

File: packages/config/config/devices/0x0441/ibt4zwave.json [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+			"$import": "#30[0xff00]",
+			"label": "Alarm Configuration - 5th Slot Notification Event Parameter",
+			"defaultValue": 0
+		},
+		"34[0xff]": {
+			"$import": "#30[0xff]",
+			"label": "Alarm Configuration - 5th Slot Action",
+			"defaultValue": 0
+		},
+
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0441/ibt4zwave.json:262]:
```suggestion
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Almost done, just some small corrections.
There seems to be a conflict. Please merge in the target branch (or rebase off it).

**Comment by cdnninja** [comment]
My pull request was from the wrong branch.   I think I fixed this with #2576.

**Pull request closed without merging**